main.py

```diff
@@ -1,28 +1,105 @@
 """
-Sample Endpoint for Qxf2 Survey Application
+FastAPI Endpoints for a Sample Message, Database Authentication,
+and adding new Nodes to the Database. 
 """
 
+import json
+import uvicorn
 from fastapi import FastAPI
-from py2neo import Graph
+from py2neo import Node, Graph
 from decouple import config
-import uvicorn
+
+# Grabbing environment variables
+HOSTNAME = config("DATABASE_HOST")
+USERNAME = config("DATABASE_USERNAME")
+PASSWORD = config("DATABASE_PASSWORD")
+
+# Authenticating with the Database
+try:
+    GRAPH = Graph(HOSTNAME, auth=(USERNAME, PASSWORD))
+    DB_FLAG = 0
+    # Deleting all the existing Nodes
+    GRAPH.delete_all()
+except:
+    DB_FLAG = 1
+
+# Adding Questions and their individual Labels
+QUESTION_LABEL = 'Question'
+
+QUESTION_1 = {
+    'label': QUESTION_LABEL,
+    'properties': {
+        'id': 1,
+        'title': 'What is your E-Mail?'
+    }
+}
+
+QUESTION_2 = {
+    'label': QUESTION_LABEL,
+    'properties': {
+        'id': 2,
+        'title': 'Who helped you this week?'
+    }
+}
+
+QUESTION_3 = {
+    'label': QUESTION_LABEL,
+    'properties': {
+        'id': 3,
+        'title': 'Whom did you help this week?'
+    }
+}
+
+QUESTION_4 = {
+    'label': QUESTION_LABEL,
+    'properties': {
+        'id': 4,
+        'title': 'What technologies did you learn?'
```

Comment:
Can you change the title to be consistent with what we have in the current survey